### PR TITLE
Between dates is always positive

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md [hunk 1]
```diff
@@ -29,19 +29,19 @@ The output is described in the table below:
 
 | Value                                                        | Type    |
 | ------------------------------------------------------------ | ------- |
-| The difference between the two dates measured in milliseconds | Decimal |
+| The difference between the two dates measured in milliseconds | Non-negative Decimal |
```
Comment (emirmx) [inline content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md]: It feels like this suggests we have a `Non-negative Decimal` data type.

File: content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md [hunk 2]
```diff
@@ -64,19 +64,19 @@ The output is described in the table below:
 
 | Value                                                     | Type    |
 | --------------------------------------------------------- | ------- |
-| The difference between the two dates measured in seconds. | Decimal |
+| The difference between the two dates measured in seconds. | Non-negative Decimal |
```
Comment (emirmx) [inline content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md]: Would it be more clear to use `absolute difference` in all places?

File: content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md]: OK - have made your other suggestion.

File: content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/application-logic/expressions/between-date-function-calls.md]: @emirmx - done this. Does it look OK to publish?

**Comment by MarkvanMents** [comment]
Approved over Slack.

**Pull request merged**